src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -45,10 +45,11 @@
      * @throws FormatException when a file has a syntax error
      * @throws PathException   when a file does not exist or is not readable
      */
-    public function load($path/*, ...$paths*/)
+    public function load($path, ...$paths)
```

Comment:
well, having `load($path, ...$paths)` means that the signature itself tells people that at least 1 argument is required (and IDEs know it too then). `load(...$paths)` allows 0 arguments